_posts/2017-06-09-the-lead-developer-conference.md

```diff
@@ -260,6 +260,44 @@ Katherine brought an interesting and new perspective of (mis)communication betwe
 
 ### Anjuan Simmons - Leadership lessons from the agile manifesto <a name="#Anjuan_Simmons">&nbsp;</a>
 
+Anjuan starts with recounting the greatest influence on his way of being a leader: The Hero with a Thousand Faces, by Joseph Campbell. This book is about the path of the hero through life. Based on the book, Christopher Vogler defines a model for the hero's path. Anjuan summarizes it as follows:
+
+1. The Call to Adventure
+2. Extreme trials
+3. Transformation
+4. Road back
+
+**The call to adventure** is often met with refusal —fear or responsibility in the ordinary life. But eventually the adventurer meets a mentor. She gives the hero both wisdom and gifts to help the hero on her journey. The hero is emboldend to overcome **extreme trials** and level up to approach the final ordeal. Solving this ordeal **transforms** the hero, often portrayed by being given a reward. Next, **the hero returns home**, atoning for the conflicts that were left behind in the ordinary world. Finally she uses her experience and the reward to improve the ordinary world.
+
+Anjuan compares this with his decision to become a software developer and the trials he had to face and the experience he had to gain. He received certain gifts like structured, logic thinking that he uses to be a better leader.
+
+The part that drew him most to Campbell's model is the hero–mentor part. He realised the mentor has completed the hero's story and is now helping someone else do it. In a way, the mentor is passing the baton to the hero.
+
+Actual baton's don't play a big role in real life, but Anjuan has found that something that comes close is **influence**. If he can improve this skill, he feels he will become a better leader.
```

Comment:
type: baton's -> _baton_

I would also note that he's passing his skill, something untangible, over to his mentee(s).